include/aspect/particle/property/crystal_preferred_orientation.h

```diff
@@ -222,6 +274,38 @@ namespace aspect
           unsigned int
           get_number_of_minerals() const;
 
+          /**
+           * @brief Determines the deformation type from the deformation type selector.
```

Comment:
Would this correctly explain this function? If so, please include. If not, please include a correct explanation ;-).

```suggestion
           * @brief Determines the deformation type from the deformation type selector.
           * If the provided @p deformation_type_selector is a specific deformation type,
           * the function will simply return the corresponding deformation type. However,
           * if the @p deformation_type_selector is an algorithm to determine the current
           * deformation type (e.g. based on measured lab data or analytical models), then
           * the function computes the appropriate deformation type at the given conditions
           * and returns the compute deformation type.
```